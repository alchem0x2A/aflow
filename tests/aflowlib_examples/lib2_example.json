{"aurl":"aflowlib.duke.edu:AFLOWDATA/LIB2_RAW/AlCa_sv/17","auid":"aflow:30b403721b9ffa88","data_api":"aapi1.2","data_source":"aflowlib","loop":["lock","thermodynamics"],"code":"vasp.4.6.35","compound":"Al2Ca2","prototype":"17","nspecies":2,"natoms":4,"natoms_orig":4,"composition":[2,2],"density":2.12557,"density_orig":1.90055,"scintillation_attenuation_length":5.52332,"stoichiometry":[0.5,0.5],"species":["Al","Ca"],"species_pp":["Al","Ca_sv"],"dft_type":["PAW_PBE"],"species_pp_version":["Al:PAW_PBE:04Jan2001","Ca_sv:PAW_PBE:06Sep2000"],"species_pp_ZVAL":[3,10],"species_pp_AUID":["423324aa45d0242f","a5df1335919961ad"],"valence_cell_iupac":10,"valence_cell_std":10,"volume_cell":104.78,"volume_atom":26.1949,"volume_cell_orig":117.185,"volume_atom_orig":29.2963,"pressure":0,"stress_tensor":[-0.74,0,0,0,-0.97,0.57,0,0.57,-1.04],"pressure_residual":-0.92,"Pulay_stress":0,"geometry":[6.14778,6.14778,5.370551,68.43991,68.43991,33.99581],"geometry_orig":[5.993357,5.993357,13.84106,150,150,33.55731],"energy_cell":-12.1373,"energy_atom":-3.03433,"energy_cutoff":373.2,"delta_electronic_energy_convergence":5.46003e-06,"delta_electronic_energy_threshold":0.0001,"kpoints_relax":[14,14,8],"kpoints":[[14,14,8]],"enthalpy_cell":-12.1373,"enthalpy_atom":-3.03433,"eentropy_cell":-3.75e-05,"eentropy_atom":-9.375e-06,"enthalpy_formation_cell":-0.648078,"enthalpy_formation_atom":-0.162019,"entropic_temperature":2712.49,"PV_cell":0,"PV_atom":0,"spin_cell":0,"spin_atom":0,"spinD":[0,0,0,0],"calculation_time":6861.88,"calculation_memory":1462.51,"calculation_cores":16,"nbondxx":[2.70644209,3.27208492,3.47185161],"sg":["C2/m #12","C2/m #12","C2/m #12"],"sg2":["C2/m #12","C2/m #12","C2/m #12"],"spacegroup_orig":12,"spacegroup_relax":12,"forces":[[0,-0.000886,-0.035644],[0,0.000886,0.035644],[0,-0.006578,0.012341],[0,0.006578,-0.012341]],"positions_cartesian":[[0,2.02365,5.12],[0,9.62008,4.58694],[0,3.54364,1.35919],[0,6.46744,3.23138]],"positions_fractional":[[0.034212046,0.034212046,0.99399257],[0.71578795,0.71578795,0.75600743],[0.27292031,0.27292031,0.21207813],[0.47707969,0.47707969,0.53792187]],"Bravais_lattice_orig":"MCLC","lattice_variation_orig":"MCLC1","lattice_system_orig":"monoclinic","Pearson_symbol_orig":"mS8","Bravais_lattice_relax":"MCLC","lattice_variation_relax":"MCLC1","lattice_system_relax":"monoclinic","Pearson_symbol_relax":"mS8","crystal_family_orig":"monoclinic","crystal_system_orig":"monoclinic","crystal_class_orig":"monoclinic-prismatic","point_group_Hermann_Mauguin_orig":"2/m","point_group_Schoenflies_orig":"C_2h","point_group_orbifold_orig":"2*","point_group_type_orig":"centrosymmetric","point_group_order_orig":4,"point_group_structure_orig":"2_x_cyclic","Bravais_lattice_lattice_type_orig":"MCLC","Bravais_lattice_lattice_variation_type_orig":"MCLC1","Bravais_lattice_lattice_system_orig":"monoclinic","Bravais_superlattice_lattice_type_orig":"FCC","Bravais_superlattice_lattice_variation_type_orig":"FCC","Bravais_superlattice_lattice_system_orig":"cubic","Pearson_symbol_superlattice_orig":"cF4","reciprocal_geometry_orig":[2.2239,2.2239,1.0646,58.518,58.518,109.47],"reciprocal_volume_cell_orig":2.11673,"reciprocal_lattice_type_orig":"MCLC","reciprocal_lattice_variation_type_orig":"MCLC3","Wyckoff_letters_orig":[["i"],["i"]],"Wyckoff_multiplicities_orig":[[4],[4]],"Wyckoff_site_symmetries_orig":[["m"],["m"]],"crystal_family":"monoclinic","crystal_system":"monoclinic","crystal_class":"monoclinic-prismatic","point_group_Hermann_Mauguin":"2/m","point_group_Schoenflies":"C_2h","point_group_orbifold":"2*","point_group_type":"centrosymmetric","point_group_order":4,"point_group_structure":"2_x_cyclic","Bravais_lattice_lattice_type":"MCLC","Bravais_lattice_lattice_variation_type":"MCLC1","Bravais_lattice_lattice_system":"monoclinic","Bravais_superlattice_lattice_type":"MCLC","Bravais_superlattice_lattice_variation_type":"MCLC1","Bravais_superlattice_lattice_system":"monoclinic","Pearson_symbol_superlattice":"mS8","reciprocal_geometry":[1.8414,1.8414,1.2672,96.937,96.937,143.36],"reciprocal_volume_cell":2.36735,"reciprocal_lattice_type":"MCLC","reciprocal_lattice_variation_type":"MCLC3","Wyckoff_letters":[["i"],["i"]],"Wyckoff_multiplicities":[[4],[4]],"Wyckoff_site_symmetries":[["m"],["m"]],"anrl_label_orig":"AB_mC8_12_i_i","anrl_parameter_list_orig":["a","b/a","c/a","beta","x1","z1","x2","z2"],"anrl_parameter_values_orig":[11.4764,0.301514,0.522237,100.025,0.625,0.875,0.875,0.625],"anrl_label_relax":"AB_mC8_12_i_i","anrl_parameter_list_relax":["a","b/a","c/a","beta","x1","z1","x2","z2"],"anrl_parameter_values_relax":[11.7584,0.305688,0.92607,152.913,0.778205,0.618993,0.234998,0.837078],"files":["AlCa_sv.17.cif","AlCa_sv.17_corner.cif","AlCa_sv.17_sconv.cif","AlCa_sv.17_sconv_corner.cif","AlCa_sv.17_sprim.cif","AlCa_sv.17_sprim_corner.cif","AlCa_sv.17_structure_relax.json","AlCa_sv.17_structure_relax1.json","CONTCAR.relax","CONTCAR.relax.abinit","CONTCAR.relax.aims","CONTCAR.relax.qe","CONTCAR.relax.vasp","CONTCAR.relax1","CONTCAR.relax2","KPOINTS.relax","LOCK","OUTCAR.relax.xz","POSCAR.orig","POSCAR.relax2","aflow.agroup.orig.json.xz","aflow.agroup.orig.out.xz","aflow.agroup.relax.json.xz","aflow.agroup.relax.out.xz","aflow.fgroup.orig.json.xz","aflow.fgroup.orig.out.xz","aflow.fgroup.relax.json.xz","aflow.fgroup.relax.out.xz","aflow.iatoms.orig.json.xz","aflow.iatoms.orig.out.xz","aflow.iatoms.relax.json.xz","aflow.iatoms.relax.out.xz","aflow.in","aflow.pgroup.orig.json.xz","aflow.pgroup.orig.out.xz","aflow.pgroup.relax.json.xz","aflow.pgroup.relax.out.xz","aflow.pgroup_xtal.orig.json.xz","aflow.pgroup_xtal.orig.out.xz","aflow.pgroup_xtal.relax.json.xz","aflow.pgroup_xtal.relax.out.xz","aflow.pgroupk.orig.json.xz","aflow.pgroupk.orig.out.xz","aflow.pgroupk.relax.json.xz","aflow.pgroupk.relax.out.xz","aflow.pgroupk_xtal.orig.json.xz","aflow.pgroupk_xtal.orig.out.xz","aflow.pgroupk_xtal.relax.json.xz","aflow.pgroupk_xtal.relax.out.xz","aflow.qmvasp.out","edata.orig.json","edata.orig.out","edata.relax.json","edata.relax.out"],"aflow_version":"aflow30680","catalog":"LIB2","aflowlib_version":"3.2.2l","aflowlib_date":"20200423_16:37:32_GMT-5"}
