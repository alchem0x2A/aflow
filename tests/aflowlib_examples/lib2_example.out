aurl=aflowlib.duke.edu:AFLOWDATA/LIB2_RAW/AgAl/8 | auid=aflow:496b9b0c3a2a92c2 | data_api=aapi1.2 | data_source=aflowlib | loop=lock,thermodynamics | code=vasp.4.6.35 | compound=Ag2Al1 | prototype=8 | nspecies=2 | natoms=3 | natoms_orig=3 | composition=2,1 | density=7.95992 | density_orig=7.6718 | scintillation_attenuation_length=1.44314 | stoichiometry=0.666666667,0.333333333 | species=Ag,Al | species_pp=Ag,Al | dft_type=PAW_PBE | species_pp_version=Ag:PAW_PBE:06Sep2000,Al:PAW_PBE:04Jan2001 | species_pp_ZVAL=11,3 | species_pp_AUID=eca65d7d992efb48,423324aa45d0242f | valence_cell_iupac=11 | valence_cell_std=25 | volume_cell=50.634 | volume_atom=16.878 | volume_cell_orig=52.5356 | volume_atom_orig=17.5119 | pressure=0 | stress_tensor=-0.33,0,0,0,-0.14,0,0,0,-0.77 | pressure_residual=-0.41 | Pulay_stress=0 | geometry=5.133046,5.133046,5.133046,144.7314,139.2352,54.98077 | geometry_orig=5.048664,5.048664,5.048664,146.4427,131.8103,60 | energy_cell=-9.5906 | energy_atom=-3.19687 | energy_cutoff=349.8 | delta_electronic_energy_convergence=6.74438e-07 | delta_electronic_energy_threshold=1e-06 | kpoints_relax=16,16,16 | kpoints=16,16,16 | enthalpy_cell=-9.5906 | enthalpy_atom=-3.19687 | eentropy_cell=0.00014725 | eentropy_atom=4.90833e-05 | enthalpy_formation_cell=-0.191662 | enthalpy_formation_atom=-0.0638873 | entropic_temperature=1164.75 | PV_cell=0 | PV_atom=0 | spin_cell=0 | spin_atom=0 | spinD=0,0,0 | stoich=0.6667   0.3333 | calculation_time=10383.4 | calculation_memory=715.71 | calculation_cores=12 | nbondxx=2.8918,2.7757,3.11 | sg=Immm #71,Immm #71,Immm #71 | sg2=Immm #71,Immm #71,Immm #71 | spacegroup_orig=71 | spacegroup_relax=71 | forces=0,0,0.00134;0,0,-0.00134;0,0,0 | positions_cartesian=0,0,5.99934;0,0,3.10759;0,0,0 | positions_fractional=0.65876663,0.65876663,0;0.34123337,0.34123337,0;0,0,0 | Bravais_lattice_orig=ORCI | lattice_variation_orig=ORCI | lattice_system_orig=orthorhombic | Pearson_symbol_orig=oI6 | Bravais_lattice_relax=ORCI | lattice_variation_relax=ORCI | lattice_system_relax=orthorhombic | Pearson_symbol_relax=oI6 | crystal_family_orig=orthorhombic | crystal_system_orig=orthorhombic | crystal_class_orig=orthorhombic-bipyramidal | point_group_Hermann_Mauguin_orig=mmm | point_group_Schoenflies_orig=D_2h | point_group_orbifold_orig=*222 | point_group_type_orig=centrosymmetric | point_group_order_orig=8 | point_group_structure_orig=2_x_dihedral | Bravais_lattice_lattice_type_orig=ORCI | Bravais_lattice_lattice_variation_type_orig=ORCI | Bravais_lattice_lattice_system_orig=orthorhombic | Bravais_superlattice_lattice_type_orig=FCC | Bravais_superlattice_lattice_variation_type_orig=FCC | Bravais_superlattice_lattice_system_orig=cubic | Pearson_symbol_superlattice_orig=cF4 | reciprocal_geometry_orig=1.6851,2.2722,2.64,39.232,58.518,82.251 | reciprocal_volume_cell_orig=4.72156 | reciprocal_lattice_type_orig=ORCF | reciprocal_lattice_variation_type_orig=ORCF1 | Wyckoff_letters_orig=e;a | Wyckoff_multiplicities_orig=4;2 | Wyckoff_site_symmetries_orig=2mm;mmm | crystal_family=orthorhombic | crystal_system=orthorhombic | crystal_class=orthorhombic-bipyramidal | point_group_Hermann_Mauguin=mmm | point_group_Schoenflies=D_2h | point_group_orbifold=*222 | point_group_type=centrosymmetric | point_group_order=8 | point_group_structure=2_x_dihedral | Bravais_lattice_lattice_type=ORCI | Bravais_lattice_lattice_variation_type=ORCI | Bravais_lattice_lattice_system=orthorhombic | Bravais_superlattice_lattice_type=ORCI | Bravais_superlattice_lattice_variation_type=ORCI | Bravais_superlattice_lattice_system=orthorhombic | Pearson_symbol_superlattice=oI6 | reciprocal_geometry=1.8879,2.1349,2.6776,44.436,52.346,83.217 | reciprocal_volume_cell=4.89888 | reciprocal_lattice_type=ORCF | reciprocal_lattice_variation_type=ORCF1 | Wyckoff_letters=e;a | Wyckoff_multiplicities=4;2 | Wyckoff_site_symmetries=2mm;mmm | anrl_label_orig=A2B_oI6_71_e_a | anrl_parameter_list_orig=a,b/a,c/a,x2 | anrl_parameter_values_orig=8.7445,0.471405,0.33333,0.666667 | anrl_label_relax=A2B_oI6_71_e_a | anrl_parameter_list_relax=a,b/a,c/a,x2 | anrl_parameter_values_relax=9.1069,0.392614,0.341499,0.341233 | ael_stiffness_tensor= | ael_compliance_tensor= | files=AgAl.8.cif,AgAl.8_corner.cif,AgAl.8_sconv.cif,AgAl.8_sconv_corner.cif,AgAl.8_sprim.cif,AgAl.8_sprim_corner.cif,AgAl.8_structure_relax.json,AgAl.8_structure_relax1.json,CONTCAR.relax,CONTCAR.relax.abinit,CONTCAR.relax.aims,CONTCAR.relax.qe,CONTCAR.relax.vasp,CONTCAR.relax1,CONTCAR.relax2,KPOINTS.relax,LOCK,OUTCAR.relax.xz,POSCAR.orig,POSCAR.relax2,aflow.agroup.orig.json.xz,aflow.agroup.orig.out.xz,aflow.agroup.relax.json.xz,aflow.agroup.relax.out.xz,aflow.fgroup.orig.json.xz,aflow.fgroup.orig.out.xz,aflow.fgroup.relax.json.xz,aflow.fgroup.relax.out.xz,aflow.iatoms.orig.json.xz,aflow.iatoms.orig.out.xz,aflow.iatoms.relax.json.xz,aflow.iatoms.relax.out.xz,aflow.in,aflow.pgroup.orig.json.xz,aflow.pgroup.orig.out.xz,aflow.pgroup.relax.json.xz,aflow.pgroup.relax.out.xz,aflow.pgroup_xtal.orig.json.xz,aflow.pgroup_xtal.orig.out.xz,aflow.pgroup_xtal.relax.json.xz,aflow.pgroup_xtal.relax.out.xz,aflow.pgroupk.orig.json.xz,aflow.pgroupk.orig.out.xz,aflow.pgroupk.relax.json.xz,aflow.pgroupk.relax.out.xz,aflow.pgroupk_xtal.orig.json.xz,aflow.pgroupk_xtal.orig.out.xz,aflow.pgroupk_xtal.relax.json.xz,aflow.pgroupk_xtal.relax.out.xz,aflow.qmvasp.out,edata.orig.json,edata.orig.out,edata.relax.json,edata.relax.out | aflow_version=aflow30350 | catalog=LIB2 | aflowlib_version=3.2.2l | aflowlib_date=20200423_12:02:55_GMT-5